### PR TITLE
docs(CONTRIBUTING): add codepen example as basis for issues

### PR DESCRIPTION
**Pull request by thgreasi**
Maybe something like this could also be added in README.
The code of this pen is based on one of the provided examples of this repo and has been used in some of the reported issues.
It obviously can be replaced by something better.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,10 @@ Please
 [check the existing issues](https://github.com/mozilla/localForage/issues) to
 make sure your issue hasn't already been filed.
 
-If you have a bug to report, please file it. If you'd like to see a feature
+If you have a bug to report, please file it.
+You are also encouradged to create an example (or [edit an existing one](http://codepen.io/thgreasi/pen/ojYKeE)) to showcase your issue.
```
Comment (tofumatt) [inline CONTRIBUTING.md]: Typo here: "encouradged" should be "encouraged". 😄 

Otherwise 👍

**Comment by tofumatt** [comment]
Good idea, r+wc, feel free to merge after the typo is fixed. 👍

**Comment by thgreasi** [comment]
Fixed that!

**Comment by thgreasi** [comment]
Good night from Greece.
Wishes for a productive new week.

**Pull request merged**